### ISSUE TITLE
Using Google Files JSON for Google Backup?

### ISSUE THREAD

**Issue by MisterTFM** [jneilliii/OctoPrint-GoogleDriveBackup#33]
I got Google Drive working about a week ago.  I updated my JSON using the new step #13.

Should the previous (April 3rd) Google Drive JSON work for a newly installed (April 10th) Google Backup Plugin?

**Comment by jneilliii** [comment]
Yes, it should in theory work for both with the updated `#13` step

**Comment by MisterTFM** [comment]
I had to create a new project and generate a new JSON secret file to get Google Backup to authorize and start backing up to my Google Drive
![ScreenHunter 336](https://user-images.githubusercontent.com/74779407/162643493-8cdf2183-f499-41fb-9508-1d2e8f00e115.jpg)
![ScreenHunter 337](https://user-images.githubusercontent.com/74779407/162643496-0fcec9be-e012-45aa-9400-f33a4f3b6e61.jpg)
.

**Comment by MisterTFM** [comment]
I had to create a new project and generate a new JSON to get Google Backup to work.  

Different URL's ???
![ScreenHunter 338](https://user-images.githubusercontent.com/74779407/162643613-91f20206-c3f2-4b1e-8fe1-285c443d367e.jpg)
![ScreenHunter 339](https://user-images.githubusercontent.com/74779407/162643629-99de997d-de25-4216-b209-da5b4332d955.jpg)

**Comment by jneilliii** [comment]
It shouldn't technically matter because they are giving the same permissions but would need to have a redirect_uri configured in the current credential setup.
